packages/wallets/babel.config.json

```diff
@@ -11,6 +11,7 @@
         "@babel/plugin-syntax-dynamic-import",
         ["@babel/plugin-proposal-unicode-property-regex", { "useUnicodeFlag": false }],
         "@babel/plugin-proposal-optional-chaining",
-        "@babel/plugin-proposal-nullish-coalescing-operator"
+        "@babel/plugin-proposal-nullish-coalescing-operator",
+        "babel-plugin-transform-barrels"
```

Comment:
Barrel files makes it harder for Webpack to tree shake, and importing one module from the barrel file includes all of the unused modules as well, this plugin adds modularize imports which transforms the following imports:
```
import { Text, Button } from "../components"
```
into:
```
import Text from "../components/Text"
import Button from "../components/Button"
```